### PR TITLE
Removes boxes from the uplink PART 1

### PR DESCRIPTION
**Pull request by Cocacolagua**
## What Does This PR Do
The nanocalcium injector no longer comes in a box with instructions, all the relevant information has been added to the uplink description / item description.
The elite MODsuit and the syndicate MODsuit no longer come in a box when bought from the uplink, the syndimask and oxytank from the box now spawn inside the MODsuit.
Also renames the combat bakery kit box to actually be "combat bakery kit" because right now its just "box" qwerty forgot to name it, mock and shame.

## Why It's Good For The Game
Next time you buy a super surplus try to count how many of the items you get come inside a box. its an EPIDEMIC. needless clutter is needless, better to not have it.
This first PR only targets the items that can have a 1:1 translation of not needing to come in a box so it can qualify as strictly quality of life, the second PR will include balance changes to remove the box.

## Testing
1. give self uplink
2. see uplink descs of items
3. spawn items
4. no box and works

## Changelog
:cl:
tweak: The nanocalcium injector, the syndicate MODsuit, and the elite MODsuit no longer come in a syndicate box.
fix: The combat bakery kit's box is now named "combat bakery kit" rather than just "box".
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -339,7 +324,7 @@
 
 <B>Usage:</B> <BR><BR><BR>
```
Comment (Burzah) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:325]:
```suggestion
<b>Usage:</b><br><br><br>
```

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -339,7 +324,7 @@
...
 
-<font size = "1">This is a highly experimental prototype chemical designed to repair damaged bones, organs, and treat interenal bleeding of soldiers in the field, use only as a last resort. The autoinjector contains prototype nanites bearing a classifed payload. The nanites will simultaneously shut down body systems whilst aiding in repair.<BR><BR><BR>Warning: Side effects can cause temporary paralysis, loss of co-ordination and sickness. <B>Do not use with any kind of stimulant or drugs. Serious damage can occur!</B><BR><BR><BR>
+<font size = "1">This is a highly experimental prototype chemical designed to fully mend limbs and organs of soldiers in the field, use only as a last resort. The autoinjector contains prototype nanites bearing a classifed payload. The nanites will simultaneously shut down body systems whilst aiding in repair.<BR><BR><BR>Warning: Side effects can cause temporary paralysis, loss of co-ordination and sickness. <B>Do not use with any kind of stimulant or drugs. Serious damage can occur!</B><BR><BR><BR>
```
Comment (Burzah) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:327]:
```suggestion
<font size="1">This is a highly experimental prototype chemical designed to fully mend limbs and organs of soldiers in the field, use only as a last resort. The autoinjector contains prototype nanites bearing a classifed payload. The nanites will simultaneously shut down body systems whilst aiding in repair.<br>br><br>Warning: Side effects can cause temporary paralysis, loss of co-ordination and sickness. <b>Do not use with any kind of stimulant or drugs. Serious damage can occur!</b><br><br><br>
```

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 3]
```diff
@@ -339,7 +324,7 @@
...
 
 To apply, hold the injector a short distance away from the outer thigh before applying firmly to the skin surface. The process of repairing should begin after a short time, during which you are advised to remain still. <BR><BR><BR><BR>After use you are advised to see a doctor at the next available opportunity. Mild scarring and tissue damage may occur after use. This is a prototype. We are not liable for any bone spurs, cancers, extra limbs, or creation of new viruses from use of the product.</font><BR><HR></font>
```
Comment (Burzah) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:329]:
```suggestion
To apply, hold the injector a short distance away from the outer thigh before applying firmly to the skin surface. The process of repairing should begin after a short time, during which you are advised to remain still. <br><br><br><br>After use you are advised to see a doctor at the next available opportunity. Mild scarring and tissue damage may occur after use. This is a prototype. We are not liable for any bone spurs, cancers, extra limbs, or creation of new viruses from use of the product.</font><br><hr></font>
```

**Review by Burzah — COMMENTED** [review]
I have a few the tags, while we are here.

File: code/datums/uplink_items/uplink_general.dm [hunk 4]
```diff
@@ -579,10 +579,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 10
 
 /datum/uplink_item/device_tools/bonerepair
-	name = "Prototype Nanite Autoinjector Kit"
-	desc = "Stolen prototype full body repair nanites. Contains one prototype nanite autoinjector and guide."
+	name = "Prototype Nanite Autoinjector"
+	desc = "Stolen prototype full body repair nanites. On injection it will shut down body systems as it revitilizes limbs and organs."
 	reference = "NCAI"
-	item = /obj/item/storage/box/syndie_kit/bonerepair
```
Comment (Henri215) [inline code/datums/uplink_items/uplink_general.dm:585]: Not going to remove that box from every place its being used on? You gave good enough descriptions for it to not need a guide anymore.

**Review by DGamerL — APPROVED** [review]
CI seems to be failing for unrelated reasons, LGTM

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**